blobstore-fs/src/main.rs

```diff
@@ -645,3 +690,32 @@ impl Blobstore for FsProvider {
         "wasmcloud:blobstore"
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::FsProvider;
+    use std::io::ErrorKind as IoErrorKind;
+    use std::path::PathBuf;
+
+    /// Ensure that only safe subpaths are resolved
+    #[test]
```

Comment:
These tests use async functions, so you'll need to use `tokio::test` as the attribute and prepend `async` for each of the test `fn` here

Reply:
Ah yup -- will fix

- [ ] replace `test` with `tokio::test`